lib/handlers/HeadHandler.js

```diff
@@ -11,7 +11,7 @@ class HeadHandler extends BaseHandler {
      * @return {function}
      */
     send(req, res) {
-        const re = new RegExp('\\' + this.store.path + '\\/(\\w+)\/?'); // eslint-disable-line prefer-template
+        const re = new RegExp('\\' + this.store.path + '\\/([\\w\-]+)\/?'); // eslint-disable-line prefer-template
```

Comment:
Yes, that was to match the example function from the issue. I think `\S` would be more concise too :)